Assets/Scripts/AI/RacingAI.cs

```diff
@@ -112,4 +117,10 @@ private Vector3 SeparationBehavior()
         return separationVelocity;
     }
 
+    IEnumerator StartRace()
+    {
+        yield return new WaitForSeconds(3f);
```

Comment:
Instead of hard coding the 3 seconds to wait maybe add the 3 seconds as a variable of the RacerManager and then use a function to get the amount of time to wait. If you need help to add this functionality to the RacerManager let me know .